tests/playbooks/tests_eth_dns_support.yml

```diff
@@ -4,122 +4,122 @@
-        nmcli connection show {{ interface }} | grep ipv4.dns
-      register: ipv4_dns
-      ignore_errors: yes
-      changed_when: false
+        - name: Verify nmcli connection DNS entry for IPv4
+          shell: |
+            set -euxo pipefail
+            nmcli connection show {{ interface }} | grep ipv4.dns
+          register: ipv4_dns
+          ignore_errors: yes
+          changed_when: false
 
-    - name: Verify nmcli connection DNS entry for IPv6
-      shell: |
-        set -euxo pipefail
-        nmcli connection show {{ interface }} | grep ipv6.dns
-      register: ipv6_dns
-      ignore_errors: yes
-      changed_when: false
+        - name: Verify nmcli connection DNS entry for IPv6
+          shell: |
+            set -euxo pipefail
+            nmcli connection show {{ interface }} | grep ipv6.dns
+          register: ipv6_dns
+          ignore_errors: yes
+          changed_when: false
 
-    - name: "Assert that DNS addresses are configured correctly"
-      assert:
-        that:
-          - "'192.0.2.2' in ipv4_dns.stdout"
-          - "'198.51.100.5' in ipv4_dns.stdout"
-          - "'2001:db8::20' in ipv6_dns.stdout"
-        msg: "DNS addresses are configured incorrectly"
+        - name: "Assert that DNS addresses are configured correctly"
+          assert:
+            that:
+              - "'192.0.2.2' in ipv4_dns.stdout"
+              - "'198.51.100.5' in ipv4_dns.stdout"
+              - "'2001:db8::20' in ipv6_dns.stdout"
+            msg: "DNS addresses are configured incorrectly"
 
-    - name: "Assert that DNS search domains are configured correctly"
-      assert:
-        that:
-          - "'example.com' in ipv4_dns.stdout"
-          - "'example.org' in ipv4_dns.stdout"
-          - "'example.com' in ipv6_dns.stdout"
-          - "'example.org' in ipv6_dns.stdout"
-        msg: "DNS search domains are configured incorrectly"
+        - name: "Assert that DNS search domains are configured correctly"
+          assert:
+            that:
+              - "'example.com' in ipv4_dns.stdout"
+              - "'example.org' in ipv4_dns.stdout"
+              - "'example.com' in ipv6_dns.stdout"
+              - "'example.org' in ipv6_dns.stdout"
+            msg: "DNS search domains are configured incorrectly"
 
-    - name: "Assert that DNS options are configured correctly"
-      assert:
-        that:
-          - "'rotate' in ipv4_dns.stdout"
-          - "'timeout:1' in ipv4_dns.stdout"
-          - "'rotate' in ipv6_dns.stdout"
-          - "'timeout:1' in ipv6_dns.stdout"
-        msg: "DNS options are configured incorrectly"
+        - name: "Assert that DNS options are configured correctly"
+          assert:
+            that:
+              - "'rotate' in ipv4_dns.stdout"
+              - "'timeout:1' in ipv4_dns.stdout"
+              - "'rotate' in ipv6_dns.stdout"
+              - "'timeout:1' in ipv6_dns.stdout"
+            msg: "DNS options are configured incorrectly"
+      vars:
```

Comment:
You can put `vars:` at the top of the block for visibility and readability, like this:
```
  tasks:
    - name: my block
      vars:
        type: veth
        interface: ethtest0
      block:
        - include_tasks...
```